index.html

```diff
@@ -145,6 +208,7 @@ <h2 class="form-header">Contact Me</h2>
                    </label>
                 <textarea name="message" id="msg" class="text-area" placeholder="Write your Message here" maxlength="500" required></textarea>
             <button type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] your contact form in the desktop version doesn't look like the one in the Figma design, I suggest sticking to the Figma design and making some changes so your contact form will be as one in the Figma design 

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/165810435-74eabb1e-7883-449b-9283-e01b7c3b2cef.png)

- [ ] Figma design
![image](https://user-images.githubusercontent.com/61790587/165810498-b3ba589f-1624-4f37-9fb7-3eb32be7aee1.png)